vnext/Desktop.IntegrationTests/HttpResourceIntegrationTests.cpp

```diff
@@ -354,6 +354,104 @@ TEST_CLASS (HttpResourceIntegrationTest) {
     Assert::AreEqual(static_cast<int64_t>(200), getResponse.StatusCode);
     Assert::AreEqual({"Redirect Content"}, content);
   }
+
+  TEST_METHOD(TimeoutSucceeds) {
+    auto port = s_port;
+    string url = "http://localhost:" + std::to_string(port);
+
+    promise<void> getPromise;
+    string error;
+    int statusCode = 0;
+
+    auto server = std::make_shared<HttpServer>(s_port);
+    server->Callbacks().OnGet = [&getPromise](const DynamicRequest &request) -> ResponseWrapper {
```

Comment:
>getPromise [](http://example.com/codeflow?start=34&length=10)

Is the lambda doing anything with this capture?

Reply:
Nothing, actually. The capture can and will be discarded.